webapp/src/components/AssetPage/SmartBadge/SmartBadge.tsx

```diff
@@ -4,13 +4,22 @@ import { SmartIcon } from 'decentraland-ui'
 import IconBadge from '../IconBadge'
 import { Props } from './SmartBadge.types'
 import './SmartBadge.css'
+import { Section } from '../../../modules/vendor/decentraland'
+import { locations } from '../../../modules/routing/locations'
+
+const SmartBadge = ({ assetType }: Props) => {
+  
+  const href = React.useMemo(() => locations.browse({
```

Comment:
We could just import `useMemo` right?

```suggestion
  useMemo(() => {
```